### PR TITLE
[v1.2] Completed storage functionality and basic UI

### PR DESCRIPTION
**Pull request by CharmaineKoh**

### CODE REVIEW

**Comment by kester-ng** [comment]
LGTM

File: src/main/java/nasa/ui/ActivityCard.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    @FXML
+    private Label priority;
+
+
+
+    public ActivityCard(Activity activity, int displayedIndex) {
+        super(FXML);
+        this.activity = activity;
+        name.setText(activity.getName().toString());
+        date.setText(activity.getDate().toString());
```
Comment (don-tay) [inline src/main/java/nasa/ui/ActivityCard.java]: @CharmaineKoh for some activity types (eg. event), there are multiple dates, how is this being handled?

Comment (kester-ng) [reply src/main/java/nasa/ui/ActivityCard.java]: oh i oversaw this, thats right, i dont think i see a way that this logic is handled, if this is not settled by 10.30pm, i gonna merge the two branches first

Comment (EdmondOng) [reply src/main/java/nasa/ui/ActivityCard.java]: Can check if is instance of Event, thn add two Label (start and end date) into it.

**Pull request merged**